src/overlays/actors/ovl_En_Dinofos/z_en_dinofos.c

```diff
@@ -242,11 +242,11 @@ static DamageTable sDamageTable = {
 
 static CollisionCheckInfoInit sColChkInfoInit = { 4, 40, 100, 80 };
 
-static TexturePtr D_8089E33C[] = {
-    object_dinofos_Tex_008E30,
-    object_dinofos_Tex_009030,
-    object_dinofos_Tex_009230,
-    object_dinofos_Tex_009030,
+static TexturePtr eyeTextures[] = {
```

Comment:
```suggestion
static TexturePtr sEyeTextures[] = {
```
Since this is a static variable, style guide says to prefix with an `s`.

Reply:
oh fair i did spot that in the object decomp example but i didnt know what the s meant thanks